### PR TITLE
update mono_repo

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
CC @mit-mit

**Comment by kevmoo** [comment]
Feel free to squash/merge and delete the branch if you're happy

File: app/mono_pkg.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ stages:
   - smoke_test:
     - group:
         - dartfmt
```
Comment (mit-mit) [inline app/mono_pkg.yaml:8]: I find these group entry names confusing, as they still use the format commands.

Can we use more general terms such as 'formatting', 'analysis', and 'tests' ?

Comment (kevmoo) [reply app/mono_pkg.yaml:8]: need to update mono_repo for that.

Remember, this was based on the travis syntax invented by natalie...~5 years ago

**Review by jonasfj — APPROVED** [review]

**Pull request merged**